src/components/ModalPortal.js

```diff
@@ -299,6 +300,14 @@ export default class ModalPortal extends Component {
     this.shouldClose = null;
   };
 
+  handleOverlayRightClick = event => {    
+      if(event.target == this.overlay)
+      {
+        event.preventDefault();
```

Comment:
We may want to allow the `this.props.onOverlayRightClick` function to call `event.preventDefault()` if they want to, for more flexibility. @diasbruno what do you think? 🤔

Reply:
Better for react-modal to not interfere in the event propagation. It would be better to let the user to handles it. So, better to just forward the event if the handle is present.